colossalai/fx/tracer/meta_patch/patched_bias_addition_module/conv.py

```diff
@@ -0,0 +1,53 @@
+import copy
+import operator
+
+import torch
+
+from ..registry import bias_addition_module
+
+
+@bias_addition_module.register(torch.nn.Conv1d)
+def non_bias_nn_conv1d(tracer, target, args, kwargs):
+    return _non_bias_nn_conv(tracer, target, args, kwargs, torch.nn.functional.conv1d)
```

Comment:
I see that all these patched function inherits the same interface. From the design point of view, we should add constraints to make sure that when a new user adds a function, it follows the same interface. One way is that we can create an abstract class to ask the developer to implements the abstract method.

Reply:
I see, I will create an abstract class.